### PR TITLE
Prepare for release to pypi (ongoing/need input)

### PR DESCRIPTION
**Pull request by pkopparla**
Addresses #5 

This is a PR to track the release to pypi. A current test release page can be found here: https://test.pypi.org/project/zonebuilder/0.1.4/

I need email addresses for @Robinlovelace and @dabreegster to be added as maintainers. The readme still needs to be updated, will do later this week. It seems the name `zonebuilder` is available on pypi so I will use that instead of `zonebuilder-py`. Discussion on any related points welcome.

### CODE REVIEW

File: README.md
Comment (Robinlovelace) [inline README.md:68]: :tada:

**Review by Robinlovelace — COMMENTED** [review]
Great work @pkopparla, looks good to me. I've made a small update with a project level email address: https://github.com/zonebuilders/zonebuilder-py/pull/8
 Overall really happy with this, thank you!

**Comment by pkopparla** [comment]
Quick question, are there any more people who should be added on as maintainers? I'm still not sure how to give commit access on pypi but I'll test it out in a bit.

**Comment by pkopparla** [comment]
Here is an updated version of the test package uploaded with documentation: https://test.pypi.org/project/zonebuilder/0.1.5/
The figure is broken on pypi because of a relative link, I will need to use a permanent link to it once it's merged to master.

File: pyproject.toml
Comment (dabreegster) [inline pyproject.toml]: dabreegster@gmail.com

**Review by dabreegster — APPROVED** [review]
> are there any more people who should be added on as maintainers?

Not that I can think of. I'm sure we could add more later if needed.

**Comment by Robinlovelace** [comment]
:tada:

**Pull request merged**
